e2e-tests/pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>dev.streamx.cli</groupId>
+    <artifactId>streamx-cli</artifactId>
+    <version>0.3.0</version>
+  </parent>
+
+  <artifactId>streamx-cli-e2e-tests</artifactId>
+
+  <name>e2e-tests</name>
+  <properties>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <dependencies>
```

Comment:
reorder dependencies - first production dependencies (used in `src/main` directory) , then test dependencies (used in `src/test`)